src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random;
+
+    ColorSupplier() {
+        random = new Random();
+    }
```

Comment:
Initialize `Random` directly in field variables or with **constructor with params.**